ntoskrnl/mm/ARM3/mdlsup.c

```diff
@@ -1672,18 +1672,36 @@ MmProtectMdlSystemAddress(IN PMDL MemoryDescriptorList,
 }
 
 /*
- * @unimplemented
+ * @implemented
  */
+_IRQL_requires_max_(APC_LEVEL)
 VOID
 NTAPI
-MmProbeAndLockProcessPages(IN OUT PMDL MemoryDescriptorList,
-                           IN PEPROCESS Process,
-                           IN KPROCESSOR_MODE AccessMode,
-                           IN LOCK_OPERATION Operation)
+MmProbeAndLockProcessPages(_Inout_ PMDL MemoryDescriptorList,
```

Comment:
Please put the parameters on a newline (as in the original suggestion, like point 6 of https://reactos.org/wiki/Coding_Style#Indentation_and_line_width )

Reply:
Done.